examples/train_pr2.py

```diff
@@ -32,5 +41,7 @@ def eval_ppo2():
 
 
 if __name__ == '__main__':
+    config = tf.ConfigProto()
+    config.gpu_options.allow_growth = True
     logging.basicConfig(level=logging.DEBUG)
-    ppo2_lstm()
+    ppo2_mlp()
```

Comment:
Is that from baseline? I remember you said that tf-agent ppo was used and many parameters are tuned.

Reply:
it is in alf depot, will send out PR shortly.